src/pybind/mgr/dashboard/frontend/src/app/ceph/nfs/nfs-form/nfs-form.component.ts

```diff
@@ -160,7 +160,8 @@ export class NfsFormComponent extends CdForm implements OnInit {
       pseudo: new UntypedFormControl('', {
         validators: [
           CdValidators.requiredIf({ protocolNfsv4: true }),
-          Validators.pattern('^/[^><|&()]*$')
+          Validators.pattern('^/[^><|&()]*$'),
```

Comment:
Thanks for the PR @avanthakkar . Can you please add a comment here about the pattern and what is being validated here?

Reply:
Well this pattern isn't changed here, I've just added min length to avoid just `/` as input.